mailbag/controller.py

```diff
@@ -56,14 +59,20 @@ def generate_mailbag(self):
         csv_portion = []
         csv_portion.append(header)
 
-
+        port = 5000
+        if not self.args.dry_run:
+            _thread.start_new_thread(helper.startServer,())
```

Comment:
Starting another thread introduces a lot of complexity. I see why its probably necessary using this method, but we should think more about any alternatives.